ui-new/css/main.scss

```diff
@@ -86,6 +101,379 @@ footer {
   padding-right: 2vh;
 }
 
+.mainv .actions .button-yes {
+  margin-left: 2.5vw;
+  margin-right: 2.4vw;
+  border-radius: 4vh;
+  padding: 2vh 3vh;
+}
+
+.mainv .actions .button-no {
+  margin-left: 2.4vw;
+  margin-right: 2.5vw;
+  border-radius: 4vh;
+  padding: 2vh 3vh;
+}
+
+.cash-out-color .mainv .actions .button-yes {
+  border: 2px solid $purple-grey;
+  background: $purple-grey;
+  color: $white;
+  @include museo-bold
```

Comment:
Description-2-bold already includes @museo-bold, was this added for any reason?

Reply:
No, looks like a mistake.

Reply:
Fixed in 3f4149e